src/components/EReceiptWithSizeCalculation.tsx

```diff
@@ -33,7 +35,8 @@ function EReceiptWithSizeCalculation(props: EReceiptWithSizeCalculationProps) {
                 style={[
                     styles.w100,
                     styles.h100,
-                    {transform: `scale(${scaleFactor}) ${styles.translateZ0.transform as string}`, transformOrigin: 'top left', aspectRatio: eReceiptAspectRatio},
+                    {transform: `scale(${scaleFactor}) ${styles.translateZ0.transform as string}`, transformOrigin: 'top left'},
+                    props.shouldUseAspectRatio && {aspectRatio: eReceiptAspectRatio},
```

Comment:
Shouldn't `shouldUseAspectRatio` be by default set to true? All previous usages had this style passed 🤔 I'm just worried it might break it in other places:
<img width="835" height="235" alt="CleanShot 2025-08-29 at 08 38 35" src="https://github.com/user-attachments/assets/d3e933b8-629e-4ec3-b3c5-1c8b758454e8" />